tests/projects/misc/SameFileBelongsToMultipleProjects/SameFileBelongsToMultipleProjects/SameFileBelongsToMultipleProjects.fsproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net452></TargetFramework>
+    <TargetFramework>net472></TargetFramework>
```

Comment:
There's a typo here that survived from earlier, the inner value is `net472>` (with a trailing `&gt;`)

Reply:
oh yeah … better fix that :-)